SQL/beestation_schema.sql

```diff
@@ -568,6 +568,14 @@ CREATE TABLE `achievement_metadata` (
 	PRIMARY KEY (`achievement_key`)
 ) ENGINE=InnoDB;
 
+DROP TABLE IF EXISTS `metacoin_item_purchases`;
```

Comment:
```suggestion
DROP TABLE IF EXISTS `SS13_metacoin_item_purchases`;
```

The BeeStation schema requires `SS13_`. I need to split this into two suggestions because GitHub wont let me combine them on account of the other comment thread.